domain/secret/state/types.go

```diff
@@ -66,6 +66,7 @@ type secretInfo struct {
 	NextRotateTime   time.Time `db:"next_rotation_time"`
 	LatestExpireTime time.Time `db:"latest_expire_time"`
 	LatestRevision   int       `db:"latest_revision"`
+	LatestRevisionID string    `db:"latest_revision_uuid"`
```

Comment:
We're using RevisionID and RevisionUUID inconsistently.
Existing code uses RevisionUUID. And new code in this PR also uses RevisionUUID (see line 127). So let's stick with RevisionUUID.

Reply:
Yeah, state layer we use UUID, but outside of state we should use ID.